satellite/satellitedb/migrate.go

```diff
@@ -18,8 +18,10 @@ import (
 	"storj.io/storj/satellite/satellitedb/pbold"
 )
 
-// ErrMigrate is for tracking migration errors
-var ErrMigrate = errs.Class("migrate")
+var (
```

Comment:
I will always disagree with more lines of code

Reply:
It's about the godoc otherwise I would never have found it, not changed it

Reply:
ah, thanks for teaching me that.

Reply:
you can see the difference in the PR comment